README.md

```diff
@@ -91,6 +97,22 @@ steps:
       configs-ignore: scala-doc-tool
 ```
 
+#### Passing additional sbt parameters.
+
+In this example you are passing some specific parameters to sbt.
+
+```yaml
+
+## in .github/workflows/dependency-graph.md
+...
+steps:
+  - uses: actions/checkout@v3
+  - uses: scalacenter/sbt-dependency-submission@v2
+    with:
+      working-directory: ./my-scala-project
+      additional-sbt-parameters: -DsomeRepository.user=programmerman -DsomeNeededSetting=cool
+```
```

Comment:
```suggestion
#### Passing additional sbt options.

In this example you are passing some additional options to sbt, for e.g. some credentials to access a private repository.

```yaml

## in .github/workflows/dependency-graph.md
...
steps:
  - uses: actions/checkout@v3
  - uses: scalacenter/sbt-dependency-submission@v2
    env:
      SBT_OPTS: '-DsomeRepository.user=programmerman -DsomeNeededSetting=cool'
```

Reply:
Oh of course that's way better, will make the changes
It's friday i guess 😛